.github/workflows/website-a11y-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: pa11y tests
+
+on: 
+  pull_request:
+    branches: [ master ]
+    paths: 
+      - 'website/**/*'
+
+jobs:
+  build:
+    name: Building site and running pa11y-ci tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout source.
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v1
+
+      - name: Install pa11y-ci dependencies 
+        run: cd website && npm install 
+
+      - name: Start dev server in background
+        run: cd website && npm run start &
+      
+      - name: Delay so the dev server can build
+        run: sleep 45
+
+      - name: Run pa11y-ci.
+        run: cd website && npm run a11y-test:dev 2>&1 | tee pa11y_output.txt
+        
+      - name: Read pa11y_output file.
+        id: pa11y_output
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: website/pa11y_output.txt
+        
+      - name: Comment on pull request.
+        uses: thollander/actions-comment-pull-request@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          message: '<details><summary>Pa11y testing results</summary>
+
+
+```${{ steps.pa11y_output.outputs.content }}```
+
+</details>'
+
+      - name: Check for pa11y failures. Still pass even if there are failures. 
```

Comment:
This isn't great but there are _too_ many errors right now, it will always fail. But at least we have the feedback for a baseline.